src/bin/tikv-ctl.rs

```diff
@@ -977,6 +979,10 @@ fn main() {
                 .takes_value(true)
                 .help("pd address"),
         )
+        .arg(
```

Comment:
Should we add it as a subcommand?

Reply:
I tried. If we use `ldb` as subcommand, we has to parse it such as `let matches = app.clone().get_matches();`. 
The format of ldb cmd is complicated and is different with the tikv-ctl, so that `get_matches` would fail and we can not continue process.

Reply:
That's why I use `fn check_run_ldb_cmd` to check key word `ldb` and reformat the `args` into the way rocksdb required.

Reply:
OK, I get it, but what does the help message look like if "ldb" is an argument, something like "tikv-ctl --ldb"?

Reply:
I have made it as subcommand but not parsed it.